packages/router/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  setupFilesAfterEnv: ['./jest.setup.js'],
```

Comment:
after reading the jest docs again, `setupFilesAfterEnv` seemed like the better setup property to use so I changed it here: https://jestjs.io/docs/configuration#setupfilesafterenv-array